jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -30,28 +34,46 @@ export type usePipelineResponse = {
 export const usePipeline = (
   ppsContext: PpsContext | undefined,
   settings: MountSettings,
-  saveNotebookMetaData: (metadata: SameMetadata) => void,
+  saveNotebookMetaData: (metadata: PpsMetadata) => void,
 ): usePipelineResponse => {
   const [loading, setLoading] = useState(false);
-  const [pipelineName, setPipelineName] = useState('');
+  const [pipeline, _setPipeline] = useState({name: ''} as Pipeline);
   const [imageName, setImageName] = useState('');
   const [inputSpec, setInputSpec] = useState('');
   const [requirements, setRequirements] = useState('');
   const [errorMessage, setErrorMessage] = useState('');
   const [responseMessage, setResponseMessage] = useState('');
   const [currentNotebook, setCurrentNotebook] = useState('None');
 
+  const setPipeline = (input: string) => {
+    if (input === '') {
+      _setPipeline({name: ''} as Pipeline);
+      return;
+    }
+    const parts = input.split(/\/(.*)/s, 2);
```

Comment:
I assume yes, but is `/` a restricted character in pipeline names?

Reply:
yeah `/` is a restricted character